chall06/dmukaliy.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python3.7
+
+import sys
+
+def error(error_message, filename):
+    print(f'{sys.argv[0]}: {filename}: {error_message}')
```

Comment:
you could have use exit() with argument, but not sure you really need to create your own function for handling error while it only print a line.